docs/subnets/reference-elastic-subnets-parameters.md

```diff
@@ -0,0 +1,224 @@
+actual fee is calculated multiplying the fee rate for the delegator reward.
+The following constraints apply:
+
+* `MinDelegationFee` must be smaller or equal to `PercentDenominator`.
+
+The `MinDelegationFee` rate applies to Primary Network as well. Its Mainnet value is $2\%$.
+
+### `MinDelegatorStake`
+
+`MinDelegatorStake` has type `uint64` and it's the minimum amount of funds required to become a delegator.
+The following constraints apply:
+
+* `MinDelegatorStake` must be larger than zero.
+
+### `MaxValidatorWeightFactor`
+
+`MaxValidatorWeightFactor` has type `uint8` and it's the factor which calculates
+the maximum amount of delegation a validator can receive. A value of 1
+effectively disables delegation. You can find more details about it in the
+Delegators Weight Checks section.
+The following constraints apply:
+
+* `MaxValidatorWeightFactor` must be larger than zero.
+
+### `UptimeRequirement`
+
+`UptimeRequirement` has type `uint32` and it's the minimum percentage of its
+staking time that a validator must be online and responsive for to receive a
+reward.
+The following constraints apply:
+
+* `UptimeRequirement` must be smaller or equal `PercentDenominator`.
+
+## Reward Formula
+
+Consider an Elastic Subnet validator which stakes a $Stake$ amount `AssetID` for $Staking\:Period$ seconds[^2].
+
+Assume that at the start of the staking period there is a $Supply$ amount of `AssetID` in the Subnet.
+The maximum amount of Subnet asset is $MaximumSupply$ `AssetID`.
+
+Then at the end of its staking period, a responsive Elastic Subnet validator
+receives a reward calculated as follows:
+
+<!-- markdownlint-disable MD013 -->
+<!-- vale off -->
+$$
+Reward = \left(MaximumSupply - Supply \right) \times \frac{Stake}{Supply} \times \frac{Staking Period}{Minting Period} \times EffectiveRate
+$$
+where
+$$
+EffectiveRate = \frac{MinConsumptionRate}{PercentDenominator} \times \left(1- \frac{Staking Period}{Minting Period}\right) + \frac{MaxConsumptionRate}{PercentDenominator} \times \frac{Staking Period}{Minting Period}
+$$
+<!-- vale on -->
+<!-- markdownlint-enable MD013 -->
+
+$Effective\:Period$ is a linear combination of $MinConsumptionRate$ and
+$MaxConsumptionRate$.
+$MinConsumptionRate$ and $MaxConsumptionRate$ bound $Effective\:Period$ because 
+
+<!-- markdownlint-disable MD013 -->
+<!-- vale off -->
+$$
+MinConsumptionRate \leq EffectiveRate \leq MaxConsumptionRate
+$$
+<!-- vale on -->
+<!-- markdownlint-enable MD013 -->
+
+The larger $Staking\:Period$ is, the closer $Effective\:Period$ is to $MaxConsumptionRate$.
+
+A staker achieves the maximum reward for its stake if $Staking\:Period$ = $Minting Period$.
+The reward is:
+
+<!-- markdownlint-disable MD013 -->
+<!-- vale off -->
+$$
+Max Reward = \left(MaximumSupply - Supply \right) \times \frac{Stake}{Supply} \times \frac{MaxConsumptionRate}{PercentDenominator}
+$$
+<!-- vale on -->
+<!-- markdownlint-enable MD013 -->
+
+## Delegators Weight Checks
```

Comment:
@dasconnor this section here is prolly not right in a reference doc and should be moved to an explaination/discussion doc. I'd ask to keep it here until I draft such a doc.